server/src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,83 @@
+import jwt from "jsonwebtoken";
+import { jwtKey } from "../config/env.js";
+
+export const checkGoogleAuth = async (req, res, next) => {
```

Comment:
Are `checkGoogleAuth` or `checkJwt` used? Dunno if I'm missing it

Reply:
yes, the google auth middleware runs on the User route, checkJwt runs on all Appointment routes, and checkAdmin runs on the two admin dashboard routes